dkist/net/attrs.py

```diff
@@ -469,3 +470,17 @@ class HeaderVersion(_SimpleAttr):
     """
     def __init__(self, version: str):
         super().__init__(version)
+
+
+#productID
```

Comment:
```suggestion
#productIds
```

This is the key in raw inventory